internal/commands/common.go

```diff
@@ -14,6 +14,7 @@ type CommonOpts struct {
 	Channel    string `short:"c" long:"channel" description:"The default snap channel to use" value-name:"CHANNEL"`
 	SectorSize string `long:"sector-size" description:"Sector size to use when creating the disk image. Only 512 and 4k sector sizes are supported." choice:"512" choice:"4096" value-name:"SECTOR-SIZE" default:"512"`
 	Validation string `long:"validation" description:"Control whether validations should be ignored or enforced" choice:"ignore" choice:"enforce"`
+	DryRun     bool   `long:"dry-run" description:"Print the states to be executed to be the image and return."`
```

Comment:
Seems like there's a typo here in the description "to be executed to be the image".

Reply:
Oh yeah, my bad -_-'